### PR TITLE
Eobs 418 patient refusal news refuse observation process

### PR DESCRIPTION
**Pull request by moderatemisbehaviour**
* New logic for partial observation with reason _refused_.
* Instead of creating an new observation due at the same time as the refused observation was originally due, the due time will be calculated from the moment of refusal based on the patient's current clinical risk. The behaviour is closer to normal observations but with the frequency capped at 24 hours.
* If a patient refuses observation straight after a patient monitoring exception a new obs will be created due in 4 hours time.

### CODE REVIEW

**Comment by Gimpneek** [comment]
:shipit:

**Comment by Gimpneek** [comment]
:shipit:

File: nh_clinical/operations.py [hunk 1]
```diff
@@ -362,6 +362,15 @@ def submit(self, cr, uid, activity_id, vals, context=None):
         return super(nh_clinical_patient_placement, self).submit(
             cr, uid, activity_id, vals, context)
 
+    @api.model
+    def get_placement_activities_for_spell(self, spell_activity_id):
+        activity_model = self.env['nh.activity']
+        domain = [
```
Comment (Gimpneek) [inline nh_clinical/operations.py]: Do you want to return all placements including those that have been done?

**Comment by Gimpneek** [comment]
:shipit:

**Comment by Nayira** [comment]
Test for story EOBS-418 and feature branch bugs EOBS-706, EOBS-709, EOBS-714, EOBS-715 and EOBS-716 have Passed. :shipit: !

File: nh_clinical/operations.py (hunk 1 above)
Comment (moderatemisbehaviour) [inline nh_clinical/operations.py]: Yeah

**Pull request merged**